### ISSUE TITLE
Yellow text and dot in the outline of a Jupyter Notebook in VS Code #209799

### ISSUE THREAD

**Issue by DavidHJong** [microsoft/vscode-jupyter#15516]
Type: <b>Bug</b>

# Behaviour
## Expected vs. Actual

In jupyter notebook, the markdown cell in outline pane should be white, but now it is yellow with a dot. 

## Steps to reproduce:

Steps to reproduce
1. Install latest vscode v1.88.0 and latest Python extension v2024.4.0.
2. Create a Jupyter Notebook with Markdown cells ## query features
3. Check the outline pane and you will see the text is yellow with a yellow dot to the right.

# Diagnostic data

-   Python version (& distribution if applicable, e.g. Anaconda): 3.11.5
-   Type of virtual environment used (e.g. conda, venv, virtualenv, etc.): Conda
-   Value of the `python.languageServer` setting: Default

<details>

<summary>Output for <code>Python</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Python</code>)
</summary>

<p>

```
XXX
```

</p>
</details>

<details>

<summary>User Settings</summary>

<p>

```

languageServer: "Pylance"

```

</p>
</details>

Extension version: 2024.2.0
VS Code version: Code 1.88.0 (5c3e652f63e798a5ac2f31ffd0d863669328dc4c, 2024-04-03T13:26:18.741Z)
OS version: Windows_NT x64 10.0.19045
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-14700KF (28 x 3418)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|191.84GB (157.64GB free)|
|Process Argv|H:\\ml4t --crash-reporter-id e65b0f0c-41c3-4df8-a3bc-a0408477b0f8|
|Screen Reader|no|
|VM|29%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscoreces:30445986
vscod805cf:30301675
binariesv615:30325510
vsaa593:30376534
py29gd2263:30899288
c4g48928:30535728
azure-dev_surveyone:30548225
2i9eh265:30646982
962ge761:30959799
pythongtdpath:30769146
welcomedialogc:30910334
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
bf62j303:30959262
cppperfnew:31000557
d34g3935:30971562
fegfb526:30981948
bg6jg535:30979843
ccp2r3:30993541
dsvsc020:30976470
pythonait:31006305
gee8j676:31009558
dsvsc021:30996838
g1icg217:30999571

```

</details>

**Comment by yuuuxt** [comment]
also on my side the outline seems frozen, showing outline of a closed notebook while editing another one. Is this related?

**Comment by Yoyokrazy** [comment]
I think there are two separate points here. The yellow text and dot are the diagnostics present in the outline, which can be controlled via the setting `outline.problems.enabled` (there are also color and badge settings that modify the behavior rather than just turning it fully on/off). If you believe that the diagnostics are showing in the outline when they shouldn't be present at all, I believe that should be filed against the provider of the diagnostics rather than the jupyter repo here.

Re: The frozen outline, that should be fixed by vscode recovery release 1.88.1. Please let me know if this reproduces even in the latest stable release of VS Code.

Thanks!
